www/source/docs/run-packages-export.html.md

```diff
@@ -21,6 +21,9 @@ You can create a Docker container image for any package by performing the follow
 
 5. You can now exit the studio. The new Docker container image exists on your computer and can be examined with `docker images` or run with `docker run`.
 
+
+Note that the `hab pkg install <local artifact>` command will only work if the studio has a copy of the origin's public key. If the install command fails with a 404 on the origin in question, make sure the public depot has a copy of the origin's public key.
```

Comment:
A good note. The default behavior for the studio is to leverage several possible configuration locations to programmatically locate the public key and  clone that key into itself. Might be better to change this to point folks towards the docs that walk people through setting up their workstation?